src/libs/actions/Report.js

```diff
@@ -600,11 +599,14 @@ function updateReportWithNewAction(
         setLocalLastRead(reportID, newMaxSequenceNumber);
     }
 
-    let messageText = lodashGet(reportAction, ['message', 0, 'text'], '');
+    let messageText = lodashGet(reportAction, ['message', 0, 'html'], '');
     if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED) {
         messageText = lodashGet(reportAction, 'originalMessage.html', '');
     }
 
+    const parser = new ExpensiMark();
+    messageText = parser.htmlToText(messageText);
+
```

Comment:
I feel like this change should be done at the backend.

Reply:
@parasharrajat  Yeah probably, any suggestion on what we should do then?

Reply:
@Luke9389 should we do the correct on the backend as well? The backend already converts the HTML to text and then sends it frontend. It does not use the text that we send in the action instead do the conversion itself. So I think we should also make the correction on the backend.

Reply:
Sorry, I was OOO last week. Just checking this one out. One moment....

Reply:
OK, yea. I agree this should be done on the backend.

Reply:
@Luke9389 @parasharrajat How do we do that?

Reply:
I think I'll have to make a PR because our backend isn't open-source. 

I'm just looking into how we prep the `text` value of that `message` key. It doesn't look like we import the `Expensimark` lib at all in that file. 

I'll come back and update you guys when that change has been deployed.